server/secrets.go

```diff
@@ -0,0 +1,84 @@
+package server
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/docker/docker/pkg/archive"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/selinux/go-selinux/label"
+	"github.com/pkg/errors"
+)
+
+// mountCfgFile holds the mount paths in the form "host:container"
+const mountCfgFile = "/etc/containers/mounts.conf"
```

Comment:
Override should be in /etc, but default should be in /usr

/usr/share/containers/mounts.conf, this is where we should ship our RHEL defaults and then allow users to override it in /etc/containers/mounts.con

Reply:
So, the code should get mount paths from both `/usr/share/containers/mounts.conf` and `/etc/containers/mounts.conf`? or should we use something like a flag or an ENV variable to know when the user wants to override the defaults?

Reply:
Override shouldn't be necessary, one could just add a line which is mounted along the other

Reply:
The standard I follow was established by systemd.  
We put content in /usr/share/APP/mounts.d/foo or /usr/lib/APP/mounts.d/foo and it gets processed.  If an admin comes in and adds /etc/APP/mounts.d/bar, then foo and bar get processed.  If the admin adds /etc/APP/mounts.d/foo, then Only the Admins foo and bar get processed.  IE He has overridden the systems foo.

We will ship a default package that mounts the RHEL secrets, this should be under /usr.  We should have a goal of nothing required to be installed in /etc. /usr == Distributions stuff.  /etc/==admin stuff. 

There is a long stated goal to make a system reset be done by executing rm -rf /etc /var; reboot
If our services can not survive that, we should make them.

Reply:
@rhatdan @runcom so what are we doing? override or no override? And if override, should `secretMounts` check the override first then the default, or should it only check the override when a flag or something is set?

Reply:
let's do the override. Basically you first use the base and only after you override.

Reply:
fixed.